docs/dev/ov_dependencies_24_2.txt

```diff
@@ -0,0 +1,374 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
```

Comment:
I would suggest adding a description of the file's content and purpose here, something like:
# this file provides a comprehensive list of all dependencies of .......
# The file is part of the automation pipeline for posting OpenVINO IR models on the HuggingFace Hub, including OneBOM dependency checks.

Reply:
Added disclaimer